src/main/java/io/debezium/connector/db2/Db2ConnectorConfig.java

```diff
@@ -226,62 +228,47 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    private static final ConfigDefinition CONFIG_DEFINITION = HistorizedRelationalDatabaseConnectorConfig.CONFIG_DEFINITION.edit()
+            .name("Db2")
+            .type(
+                    HOSTNAME,
+                    PORT,
+                    USER,
+                    PASSWORD,
+                    DATABASE_NAME)
+            .connector(
+                    SNAPSHOT_MODE,
+                    BINARY_HANDLING_MODE,
+                    INCREMENTAL_SNAPSHOT_CHUNK_SIZE,
+                    INCREMENTAL_SNAPSHOT_ALLOW_SCHEMA_CHANGES)
+            .excluding(
+                    SCHEMA_WHITELIST,
+                    SCHEMA_INCLUDE_LIST,
+                    SCHEMA_BLACKLIST,
+                    SCHEMA_EXCLUDE_LIST,
+                    // additional fields
+                    BINARY_HANDLING_MODE,
```

Comment:
I'd keep only these fields on the exclusion list

* BINARY_HANDLING_MODE
* INCLUDE_SCHEMA_COMMENTS
* INCREMENTAL_SNAPSHOT_ALLOW_SCHEMA_CHANGES
* SNAPSHOT_MAX_THREADS
* HEARTBEAT_ACTION_QUERY

The others should be supported

Reply:
Yes @jpechane , I have updated the list.